core/templates/core/post-header.html

```diff
@@ -4,11 +4,10 @@
 <div class="card-header">
     <div class="row align-items-center">
         <div class="col-auto">
-            <!-- TODO: Update avatar src with user avatar URL after merging feature/avatar -->
-            <img src="https://placehold.co/32x32.png"
-                 alt="{{ post.author }} avatar image"
-                 class="rounded-circle"
-                 style="width: 32px; height: 32px;">
+             <img src="{% if post.author.avatar %}{{ post.author.avatar.url }}{% else %}{{ default_avatar_url }}{% endif %}"
```

Comment:
Just add default to django model, it should do the trick. We don't want to repeat this all over the place.
If default on model is not working, set property on author model that will return default if avatar not set.

Reply:
Okey